libs/platform/src/lib/components/upload-collection/dialogs/move-to/move-to.component.ts

```diff
@@ -0,0 +1,125 @@
+import { Component, Inject, OnInit } from '@angular/core';
+
+import { DIALOG_REF, DialogRef, DialogService, DialogConfig } from '@fundamental-ngx/core';
+
+import { UploadCollectionItem, UploadCollectionFolder } from '../../models/upload-collection.models';
+import { NewFolderComponent } from '../new-folder';
+
+function isObject<T>(item: T): boolean {
+    return typeof item === 'object' && !Array.isArray(item) && item !== null;
+}
+
+@Component({
+    templateUrl: './move-to.component.html',
+    styles: [`
+        .fd-list__item--alternative-background {
+            background: #f2f2f2;
+        }
+
+        .fd-list__icon--alternative-background {
+            color: #74777A !important;
+        }
```

Comment:
Do we need to hardcode the colors here? Why do these need to be changed from the default colors?

Reply:
fixed